### PR TITLE
Version 0.3.1

### PR DESCRIPTION
**Pull request by filipi87**
Version 0.3.1:
- Fixed the example inside rtvi-client-react-native-daily to match the new API.
- Upgraded to use Expo 52
- Improved the readme and created env template.
- Renaming the package name.
- Upgrading to use @pipecat-ai/client-js 0.3.1
- Implementing the new methods to handle the speaker.
- Upgrading type definitions

### CODE REVIEW

File: example/app.json
Comment (filipi87) [inline example/app.json:29]: Need to remove this.

**Comment by filipi87** [comment]
Once we merge this PR, we will need to rename the repo:
`rtvi-client-react-native-daily` → `pipecat-client-react-native`

File: example/app.json
Comment (filipi87) [reply example/app.json]: Done.

File: package.json
Comment (filipi87) [inline package.json:2]: The new package name.

File: example/src/App.tsx
Comment (markbackman) [inline example/src/App.tsx:12]: How is the DailyVoiceClient used? Is that not replaced by the RTVIClient?

File: src/types/realtime-ai/typeOverrides.d.ts
Comment (markbackman) [inline src/types/realtime-ai/typeOverrides.d.ts:150]: Totally naive question: This looks like a lot of duplication from `@pipecat-ai/client-js`. Are all of these overrides or could we actually use the types from the `@pipecat-ai/client-js`?

Also, totally minor point, the directory for this file is still called `realtime-ai`. Maybe that should change now that realtime-ai is no longer a thing?

**Comment by markbackman** [comment]
Generally looks good! Just a few questions from me.

File: example/src/App.tsx
Comment (markbackman) [reply example/src/App.tsx:12]: I think that makes sense. Then the transport package is just about providing the transport. That's nice and clean.

File: src/types/realtime-ai/typeOverrides.d.ts
Comment (filipi87) [reply src/types/realtime-ai/typeOverrides.d.ts:150]: We don't need to change the directory because what actually matters is the path inside tsconfig:
https://github.com/pipecat-ai/rtvi-client-react-native-daily/blob/0.3.0/tsconfig.json#L6.

Unfortunately, we cannot override only specific types in this case. Some core types, like `MediaDeviceInfo`, `MediaStreamTrack`, are provided by `react-native-webrtc` instead of the browser, so we must override everything. We do the same with `react-native-daily-js` and `daily-js`.

Comment (filipi87) [reply src/types/realtime-ai/typeOverrides.d.ts:150]: But I can rename the repo only in order to keep consistency. 👍

File: example/src/App.tsx
Comment (filipi87) [reply example/src/App.tsx]: Done.

File: src/types/realtime-ai/typeOverrides.d.ts
Comment (filipi87) [reply src/types/realtime-ai/typeOverrides.d.ts:150]: Done.

**Review by markbackman — APPROVED** [review]
This LGTM. I'm not sure if you want to wait for a review from others, but I endorse moving forward.

**Comment by filipi87** [comment]
Hi @markbackman , I can wait until tomorrow morning, to see if I receive any more feedbacks. If not, I will move forward and publish the library.
Thanks a lot for the review. 🙂

**Pull request merged**
